spec/rollbar/middleware/js_spec.rb

```diff
@@ -106,6 +106,7 @@
 
         before do
           Object.const_set('SecureHeaders', Module.new)
+          SecureHeaders.const_set('VERSION', '3.0.0')
```

Comment:
These tests are ok, but the implementation doesn't check the version of SecureHeaders, no?